cmd/ocm/cluster/create/create.go

```diff
@@ -55,6 +55,8 @@ func init() {
 	fs := Cmd.Flags()
 	arguments.AddParameterFlag(fs, &args.parameter)
 	arguments.AddHeaderFlag(fs, &args.header)
+
+	arguments.AddProviderFlag(fs, &args.provider)
```

Comment:
we dropped support for `cluster create` command

Reply:
dropped this file from the PR.